WordPress/src/main/java/org/wordpress/android/GCMMessageService.java

```diff
@@ -207,12 +210,46 @@ private void showGroupNotificationForActiveNotificationsMap(int pushId, String n
 
     private void addActionsForCommentNotification(NotificationCompat.Builder builder, String noteId) {
         // Add some actions if this is a comment notification
-        Intent commentReplyIntent = new Intent(this, WPMainActivity.class);
-        commentReplyIntent.putExtra(WPMainActivity.ARG_OPENED_FROM_PUSH, true);
-        commentReplyIntent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_NEW_TASK
-                | Intent.FLAG_ACTIVITY_CLEAR_TASK);
-        commentReplyIntent.setAction("android.intent.action.MAIN");
-        commentReplyIntent.addCategory("android.intent.category.LAUNCHER");
+
+        boolean bActionsSet = false;
```

Comment:
The only prefix we use is `m` for member variables. Please rename this boolean to a question format, so `areActionsSet`